liquidhaskell-boot/src/Language/Haskell/Liquid/Bare.hs

```diff
@@ -1258,6 +1287,30 @@ makeMeasEnv env tycEnv sigEnv specs = do
     embs          = Bare.tcEmbs        tycEnv
     name          = Bare.tcName        tycEnv
 
+-------------------------------------------------------------------------------------------
+--- Add the opaque reflections to the measure environment
+--- Returns a new environement that is the old one enhanced with the opaque reflections
+-------------------------------------------------------------------------------------------
+addOpaqueReflMeas :: Bare.Env -> Bare.TycEnv -> Bare.MeasEnv -> Bare.ModSpecs ->
+               [(Ghc.Var, LocSpecType, F.Equation)] ->
+               Bare.Lookup Bare.MeasEnv
+-------------------------------------------------------------------------------------------
+addOpaqueReflMeas env tycEnv measEnv specs eqs = do
+  (measures0, opaqueRefl)   <- Bare.makeOpaqueReflMeasures env measEnv tycEnv specs eqs
+  -- Rest of the code is the same idea as for makeMeasEnv, only we just care on how to get
+  -- `meSyms` (no class, data constructor or other stuff here).
+  let measures = mconcat measures0
+  let (_, ms) = Bare.makeMeasureSpec'  (typeclass $ getConfig env)   measures
+  let cms      = Bare.makeClassMeasureSpec measures
```

Comment:
Is `cmeasMap measures` ever going to be non-empty? Otherwise, this call seems useless.